packages/server/src/addresses/api/AddressDto.ts

```diff
@@ -0,0 +1,38 @@
+import { ApiProperty, OmitType } from '@nestjs/swagger';
+
+import { ApiObjectIdProperty } from '../../shared';
+import { ADDRESS_CONSTANTS } from '../database';
+
+class AddressDto {
+  @ApiObjectIdProperty()
+  readonly id: string;
+
+  @ApiProperty()
+  readonly street: string;
+
+  @ApiProperty({ pattern: ADDRESS_CONSTANTS.STREET_NUMBER.REGEX.toString().slice(1, -1) })
```

Comment:
Możesz użyć [source prop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/source) (będzie krócej)
```suggestion
@ApiProperty({ pattern: ADDRESS_CONSTANTS.STREET_NUMBER.REGEX.source })
```